### PR TITLE
Pre select filters via url

### PR DESCRIPTION
**Pull request by martypowell**
- These changes target the filter controller
- The database mockup page has been updated with the new changes

_Note_: There are sections of the site that require this update, but in effort to keep this smaller. I am pushing now.

### CODE REVIEW

File: js/apps/filter-page/controllers/filterPageCtrl.js [hunk 1]
```diff
@@ -1,25 +1,25 @@
 ((app) => {
 	'use strict';
 
-	const FilterPageCtrl = function FilterPageCtrl($scope, cardService, filterService, $animate, $timeout, CONSTANTS) {
-		const self = this;
+	const FilterPageCtrl = function FilterPageCtrl($location, $scope, cardService, filterService, $animate, $timeout, CONSTANTS) {
+		const vm = this;
```
Comment (ascire) [inline js/apps/filter-page/controllers/filterPageCtrl.js]: What does `vm` mean in this context?

Comment (ascire) [reply js/apps/filter-page/controllers/filterPageCtrl.js]: Oh wait, I think I got it.  Might want to spell out `viewModel`.

File: js/apps/filter-page/controllers/filterPageCtrl.js [hunk 2]
```diff
@@ -135,15 +135,63 @@
 		angular.element(document).on('hide.bs.collapse', '.expando-wrapper .collapse', toggleIcon);
 		angular.element(document).on('show.bs.collapse', '.expando-wrapper .collapse', toggleIcon);
 
-		cardService.get(loadCardsAndFilters);
+		const formatKeyName = (key) => {
+			if (!key) return "";
+
+			return key.replace("-", " ");
+		};
```
Comment (ascire) [inline js/apps/filter-page/controllers/filterPageCtrl.js]: Hey, you like functional programming, right? ;-)

```
const formatKeyName = (key) => !key ? "" : key.replace("-", " ");
```

File: js/apps/filter-page/directives/filter.js [hunk 3]
```diff
@@ -11,7 +11,7 @@
 			if (inputType === 'radio') {
 				$input.prop('name', $scope.filterFamily.name);
 			}
-
+
```
Comment (ascire) [inline js/apps/filter-page/directives/filter.js]: Intentional whitespace?

File: js/apps/filter-page/controllers/filterPageCtrl.js (hunk 1 above)
Comment (martypowell) [reply js/apps/filter-page/controllers/filterPageCtrl.js]: yea, it's kinda the defactor standard we have been using for `viewModel` in angular

File: js/apps/filter-page/controllers/filterPageCtrl.js (hunk 2 above)
Comment (martypowell) [reply js/apps/filter-page/controllers/filterPageCtrl.js]: very nice comment, adjusted

File: js/apps/filter-page/directives/filter.js (hunk 3 above)
Comment (martypowell) [reply js/apps/filter-page/directives/filter.js]: @ascire yes, this probably isn't the greatest layout, but i think we prefer to keep white space between conditions. also between methods, unless they are one liners...something we should probably iron out though.

Comment (ascire) [reply js/apps/filter-page/directives/filter.js]: The thing that struck me was it appears to have changed from an empty line to 6 tabs.  You know there are [languages](https;//www.python.org) where this [makes a difference](https://en.wikipedia.org/wiki/Whitespace_(programming_language)), right?

Comment (martypowell) [reply js/apps/filter-page/directives/filter.js]: I see, yea thats not right.

**Comment by ascire** [comment]
Hey we fixed a bug!  I think it's good now.

**Pull request merged**